enhancements/sig-architecture/34-work-executor-group/README.md

```diff
@@ -0,0 +1,219 @@
+# Work Executor Group
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [website](https://github.com/open-cluster-management-io/website/)
+
+## Summary
+
+The proposal adds a new field `executorSubject` to the spec of `ManifestWork` 
+api, to support dynamic identity impersonation upon the execution of 
+`ManifestWork` resources. If the `executorSubject` is absent, it means no
+impersonation will be performed when the work agent applies resources to the
+spoke cluster which is also for the backward compatibility.
+
+## Motivation
+
+### Auditing the API request history from work agent
+
+Currently all the API requests raised by work agent in the managed cluster are
+from a fixed service account ("open-cluster-management-agent/klusterlet-work-sa"
+under the default installation). However in some cases, we need to clarify the
+source/owner identity of the `ManifestWork` resource so that we can record the
+api audits in a finer granularity. This will be helpful for the managed 
```

Comment:
How firm a requirement is this?  Impersonation is a high privilege, but your second motivation could be achieved similar to the openshift templateinstance API.  Only I would suggest constraining yourself to service accounts.

Reply:
in the latest design, i completely removed impersonation in favor of client-side authorization